src/energyplus/Test/ZoneHVACTerminalUnitVariableRefrigerantFlow_GTest.cpp

```diff
@@ -235,6 +256,58 @@ TEST_F(EnergyPlusFixture, ForwardTranslatorZoneHVACTerminalUnitVariableRefrigera
 
     EXPECT_EQ(idf_fan.getString(Fan_OnOffFields::AirOutletNodeName).get(), idf_supHC.getString(Coil_Heating_WaterFields::AirInletNodeName).get());
   }
+
+  // Supplemental HC = CoilHeatingWater, BlowThrough
```

Comment:
New FT test case to check that in BlowThrough, the nodes are also connected properly.